File: langgraph/data/semantic_equiv/123.py
def extract_odd_collatz_values(start_number):
    if start_number % 2 == 0:
        odd_numbers = []
    else:
        odd_numbers = [start_number]

    current_value = start_number
    while current_value > 1:
        if current_value % 2 == 0:
            current_value //= 2
        else:
            current_value = current_value * 3 + 1

        if current_value % 2 == 1:
            odd_numbers.append(current_value)

    return sorted(odd_numbers)

# method name: get_odd_collatz -> extract_odd_collatz_values
# parameter name: n -> start_number
# variable name: odd_collatz -> odd_numbers
# replaced n = n/2 with n //= 2 for integer division
# used current_value instead of n inside the loop for clarity

File: langgraph/result/qwen/14b-py2js/123.js
function getOddCollatz(n) {
    let oddCollatz;
    if (n % 2 === 0) {
        oddCollatz = [];
    } else {
        oddCollatz = [n];
    }
    while (n > 1) {
        if (n % 2 === 0) {
            n = n / 2;
        } else {
            n = n * 3 + 1;
        }
        
        if (n % 2 === 1) {
            oddCollatz.push(parseInt(n));
        }
    }
    return oddCollatz.sort((a, b) => a - b);
}